src/Behat/Gherkin/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -32,7 +39,7 @@ static function ($line) use ($resource) {
     }
 
     /**
-     * @return FeatureNode|null
+     * @return null|FeatureNode
```

Comment:
I would rather keep `null` last rather than first

Reply:
I prefer it first because:
1. in some cases (e.g. `@param`), it provides nice alignment without any special adjustments
2. the 'old' nullable type was also before (`?Something`)

Do you have any particular reasons for that preference?

Reply:
1. I don't really care about alignment. I actually disable the alignment rule in my own projects, which avoids multiline diffs for nothing (same reason why PSR-12 does not require aligning successive assignments or array values btw) and could lead to huge alignment spaces when using some long array shape in one parameter.
2. many projects (including the Symfony coding standards that you used as basis) have a rule to keep `null` at the end rather than at the start, making it more common.

Reply:
I agree, we've also disabled alignment in our projects and overall it makes life a lot easier, especially reviewing PRs.

Reply:
I generally don't like (empty-space) alignment either, but sometimes it's hard to figure out which existing rules to keep or not. For instance, the parameter type order was quite a surprise to me.